distribution/ltp/lite/configs/RHELKT1LITE.20180926

```diff
@@ -1487,12 +1558,17 @@ mm02 mmap001
 mtest01 mtest01 -p80
 mtest01w mtest01 -p80 -w
 
-mtest05 mmstress
-
-mtest06 mmap1 -x 0.05
+#test for race conditions
+mtest05   mmstress
+mtest06   mmap1 -x 0.05
```

Comment:
Since we disabled the test run today for the old version, can we do it here too, so we don't have to add that commit on top of this?